### ISSUE TITLE
Compile Error: Expecting expression but encountered '­'

### ISSUE THREAD

**Issue by GWRon** [bmx-ng/bcc#441]
While checking stuff for "cleaning" within the maxide repository I tried to compile "monk.bmx" with NG and received the error: `Compile Error: Expecting expression but encountered '­'`

The line of the error is:
```BlitzMax
	Method Create:TConnection(path$,client:TSocket)
		DebugLog "creating connection!"
'this one:
		­_serverpath=path
		_socket=client
		_stream=CreateSocketStream(client)
		connections.AddLast Self
		id=connections.count()
		Return Self
	End Method
```
_serverpath is of type string
path if of type string.

So what is BCC trying to tell me here?

**Comment by HurryStarfish** [comment]
Ooh, this is a fun one :slightly_smiling_face: 

You've removed the file by now, but after checking out the file from the commit before you did and trying to compile it, I see a "-" in front of `_serverpath` on that line. When I try to compile it, bcc tells me that too. Except that's not actually a "-". That's a byte with the value 173, which is neither standard ASCII, nor a valid first byte for an UTF-8 character. Browsers like Firefox, Chrome or IE seem to simply not display it, which is why it's invisible when the file is opened in the browser or viewed on github. Notepad, Notepad++ and MaxIDE on the other hand all render it as a "-" on my end. I can even copy/paste the `Compile Error` message and code from your post up there to any of those programs and get it to show up in the same way.

tl;dr: The file is broken.

**Comment by GWRon** [comment]
Utf8 released a bunch of nasty bugs in its early days...and is still doing ;-)

Good catch
